gemstone/run.sh

```diff
@@ -92,16 +92,16 @@ gemstone::prepare_stone() {
         mkdir "${SMALLTALK_CI_VMS}/Pharo-3.0"
         print_info "Downloading Pharo-3.0 vm to cache" 
         pushd "${SMALLTALK_CI_VMS}/Pharo-3.0" > /dev/null
-          pharo_zeroconf="$(download_file "get.pharo.org/vm30")" || print_error_and_exit "Pharo-3.0 vm download failed."
-          bash -c "${pharo_zeroconf}"  || print_error_and_exit "Pharo-3.0 vm download failed."
+          download_file "get.pharo.org/vm30" "$(pwd)/zeroconfig"
+          bash "$(pwd)/zeroconfig"
         popd > /dev/null
```

Comment:
I was out of town over the weekend, but now I'm back ... I am concerned that an error during download_file or running the zeroconfig script will not cause the run.sh script to fail and that will mean that it will be difficult to understand what actually failed ...